src/relay/op/op_common.h

```diff
@@ -84,6 +85,7 @@ namespace relay {
       .add_type_rel("Broadcast", BroadcastRel)                                          \
       .set_attr<TOpPattern>("TOpPattern", kBroadcast)                                   \
       .set_attr<TOpIsStateful>("TOpIsStateful", false)                                  \
+      .set_attr<CommutativeOp>("CommutativeOp", isCommutativeOp(OpName))                \
```

Comment:
What if it's non-comm by default (so no attrib binding), and the registration of (for now) add & multiply explicitly set it to true.